### PR TITLE
{lib}[foss/2021a] PyTorch-Ignite v0.4.9 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node134.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/29fb18c71fcbfe875c8d8b4225bff045 for a full test report.

File: easybuild/easyconfigs/p/PyTorch-Ignite/PyTorch-Ignite-0.4.9-foss-2021a-CUDA-11.3.1.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('Python', '3.9.5'),
+    ('PyTorch', '1.10.0', versionsuffix),
+]
+
+exts_list = [
+    ('pytorch-lightning', version, {
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/PyTorch-Ignite/PyTorch-Ignite-0.4.9-foss-2021a-CUDA-11.3.1.eb:21]: surely this should be
```suggestion
    ('pytorch-ignite', version, {
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in total)
node401.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, AMD EPYC 7282 16-Core Processor (zen2), 1 x NVIDIA NVIDIA A100-PCIE-40GB, 470.82.01, Python 2.7.5
See https://gist.github.com/a8f08d65ef3a5b7ea69b7a7dff674865 for a full test report.

File: easybuild/easyconfigs/p/PyTorch-Ignite/PyTorch-Ignite-0.4.9-foss-2021a-CUDA-11.3.1.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/p/PyTorch-Ignite/PyTorch-Ignite-0.4.9-foss-2021a-CUDA-11.3.1.eb:21]: whoops :)

**Review by jfgrimm — DISMISSED** [review]
probably worth switching to `PythonPackage` since it's currently only a single extension?

**Comment by smoors** [comment]
> probably worth switching to `PythonPackage` since it's currently only a single extension?

actually, I prefer to always use `PythonBundle` as it's easier to add extensions later on (which is bound to happen at some point).
do you know if there's any advantage of using `PythonPackage` besides "consistency"?

**Review by jfgrimm — COMMENTED** [review]
As decided in issue #16330, we have deprecated the use of `True` to signify a system-toolchain dependency (#16384), in favour of the more intuitive `SYSTEM` template constant. Due to the change in the test suite, please run `eb --sync-pr-with-develop 15491` and update the PR to use `SYSTEM` instead.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0103u14a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6330 CPU @ 2.00GHz (icelake), 1 x NVIDIA NVIDIA A30, 520.61.05, Python 3.6.8
See https://gist.github.com/f5f9813fcb842a1c032865fa2c780a09 for a full test report.

**Review by branfosj — APPROVED** [review]

**Pull request merged**